conans/client/graph/spdx.py

```diff
@@ -0,0 +1,58 @@
+
+def generate_sbom(conanfile, **kwargs):
+    name = conanfile.ref.name
+    version = conanfile.ref.version
+    date = datetime.fromtimestamp(time.time(), tz=timezone.utc).strftime('%Y-%m-%dT%H:%M:%SZ')
+    packages = []
+
+    for dependency in [conanfile] + conanfile.dependencies.values():
+        packages.append(
+            {
+                "name": dependency.ref.name,
+                "SPDXID": f"SPDXRef-{dependency.ref}",
+                "version": str(dependency.ref.version),
+                "license": dependency.license if dependency.license in spdx_licences else "NOASSERTION",
```

Comment:
If this is just a check that the Conan licenses  are correct, most likely we can skip it and assume they are already correct.